src/transaction.js

```diff
@@ -327,14 +345,24 @@ class Transaction {
       transaction: this._transactionId,
     };
 
-    return this._firestore.request('rollback', request);
+    return this._firestore.request('rollback', request, this._requestTag);
+  }
+
+  /**
+   * Returns the tag to use with with all request for this Transaction.
+   * @private
+   * @return {string} A unique client-generated identifier for this transaction.
+   */
+  get requestTag() {
```

Comment:
Are we ok adding this to the public API like this?

Reply:
This follows the existing pattern. As you know from breaking you last time, I still need to resolve this and expose different types publicly so that users don't have access to our internal API. Right now, the distinction is in whether something is documented in the API docs/in the .d.ts file.